client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/protocol/InternalFlag.java

```diff
@@ -0,0 +1,26 @@
+package org.infinispan.client.hotrod.impl.protocol;
+
+/**
+ * Flags which are internal to the protocol and shouldn't be exposed to users.
```

Comment:
@galderz we can move the lifespan and maxIdle values in the body, if that's your only concern :)

On a more serious note, I would really hate to send two 64-bit numbers in each write request that doesn't need them. I would maybe be ok with two 32-bit numbers, but that would still require some extra logic to change the unit automatically, in order to avoid overflows.